src/coreclr/vm/frames.cpp

```diff
@@ -198,19 +445,19 @@ const size_t FRAME_TYPES_COUNT =
 //-----------------------------------------------------------------------
 // Implementation of the global table of names.  On the DAC side, just the global pointer.
 //  On the runtime side, the array of names.
-    #define FRAME_TYPE_NAME(x) {x::GetMethodFrameVPtr(), #x} ,
+    #define FRAME_TYPE_NAME(x) {FrameIdentifier::x, #x} ,
```

Comment:
`FrameIdentifier` does not need to be in the table anymore. We can just index into the table.